docs/develop-branch.md

```diff
@@ -0,0 +1,37 @@
+develop branch
+===
+
+Building the full Windows App SDK package involves multiple repos and build pipelines.
```

Comment:
The other option here is that we have a full private fork of WASDK's repo, where there is another "main" that is only buildable by MS people.  That main is treated like this main in all the normal ways.  At the point we're ready to make internal-only changes visible (because we've published the packages) we create a PR from wasdk-internal/main to wasdk-public/main, merging in the usual way.

Upside is that the public doesn't have to see our sausage-making.

Downside is 2x the build configurations and hosting yet another repo.